.github/workflows/build.yml

```diff
@@ -25,7 +28,7 @@ jobs:
       - name: Gradle Wrapper Validation
         uses: gradle/wrapper-validation-action@v1.1.0
 
-      # Set up Java environment for the next steps
+          # Set up Java environment for the next steps
```

Comment:
This is definitely an unnecessary action

Reply:
Fixed